libevmasm/Assembly.cpp

```diff
@@ -242,13 +245,153 @@ string Assembly::toStringInHex(u256 _value)
 	hexStr << std::uppercase << hex << _value;
 	return hexStr.str();
 }
+vector<Json::Value> Assembly::assemblyItemAsJSON(AssemblyItem const& _item, int _sourceIndex) const
```

Comment:
Sadly this need to be a vector, because `Tag` is emitting two json objects, `tag` and `JUMPDEST`.